packages/rxjs/src/internal/Observable.ts

```diff
@@ -341,7 +883,7 @@ export class Observable<T> implements Subscribable<T> {
    * in the order they were passed in.
    */
   pipe(...operations: UnaryFunction<any, any>[]): unknown {
-    return pipeFromArray(operations)(this);
+    return operations.reduce(pipeReducer, this as any);
```

Comment:
Refactored more than a move: This is so small, I couldn't see a reason not to just inline it for now. the `pipeFromArray` was used in `Observable#pipe`, `pipe`, and `rx`, and we'll need to separate it up when they're in different packages.